datajoint/declare.py

```diff
@@ -311,17 +312,18 @@ def declare(full_table_name, definition, context):
         external_stores,
     ) = prepare_declare(definition, context)
 
-    metadata_attr_sql = [
-        "`_{full_table_name}_timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP"
-    ]
-    attribute_sql.extend(
-        attr.format(
-            full_table_name=sha1(
-                full_table_name.replace("`", "").encode("utf-8")
-            ).hexdigest()
+    if config.get("enable_hidden_attributes", False):
```

Comment:
How about changing to `add_hidden_timestamp` to make this feature more specific. Hidden attributes will always be enabled. But the default timestamp will be configurable.